### ISSUE TITLE
Documentation cleanup

### ISSUE THREAD

**Issue by rabitt** [mir-dataset-loaders/mirdata#138]
There are documentation mistakes all over the library - let's do an audit.

**Comment by keunwoochoi** [comment]
To update -
```
Parameters
-----------
```
to
```
Args:
```

**Comment by keunwoochoi** [comment]
and other formatting issues of docstrings.

**Comment by keunwoochoi** [comment]
in `medleydb_melody.py`, `class Track(object):` has some unexplained attributes. Some of them are probably easily guessed, but I don't wanna rely on my guess to documentation. Maybe someone can just reply here?

```
        is_excerpt (bool):
        is_instrumental (bool):
        n_sources (int): 
        melody1 (F0Data):
        melody2 (F0Data):
        melody3 (F0Data):
```

**Comment by keunwoochoi** [comment]
in `orchset`, 
```
        work (str):
        predominant_melodic_instruments (str):
        alternating_melody:
        contains_winds (bool): If the track contains any wind instrument
        contains_strings (bool): If the track contains any string instrument
        contains_brass (bool): If the track contains any brass instrument
        only_strings (bool): If the track contains string instruments only
        only_winds (bool): If the track contains wind instruments only
        only_brass (bool): If the track contains brass instruments only
```
I guessed some of them. Please correct me if it's wrong & let me know what to fill other stuff with.

**Comment by keunwoochoi** [comment]
Salami:

```
    Attributes:
        track_id (str): track id
        audio_path (str): audio path of the track
        source
        annotator_1_id
        annotator_2_id
        duration_sec: duration of the track
        title: Title of the track
        artist
        annotator_1_time
        annotator_2-time
        broad_genre
        genre:
        sections_annotator_1_uppercase: annotation
        sections_annotator_1_lowercase: annotation
        sections_annotator_2_uppercase: annotation
        sections_annotator_2_lowercase: annotation

```

I'm lost here as well. Fortunately this is the last one.

**Comment by rabitt** [comment]
> in `orchset`,
> 
> ```
>         work (str):
>         predominant_melodic_instruments (str):

list of strings

>         alternating_melody:

bool

>         contains_winds (bool): If the track contains any wind instrument
>         contains_strings (bool): If the track contains any string instrument
>         contains_brass (bool): If the track contains any brass instrument
>         only_strings (bool): If the track contains string instruments only
>         only_winds (bool): If the track contains wind instruments only
>         only_brass (bool): If the track contains brass instruments only
> ```
> 
> I guessed some of them. Please correct me if it's wrong & let me know what to fill other stuff with.

**Comment by rabitt** [comment]
> Salami:
> 
> ```
>     Attributes:
>         track_id (str): track id
>         audio_path (str): audio path of the track
>         source
>         annotator_1_id
>         annotator_2_id
>         duration_sec: duration of the track
>         title: Title of the track
>         artist
>         annotator_1_time
>         annotator_2-time
>         broad_genre
>         genre:
>         sections_annotator_1_uppercase: annotation
>         sections_annotator_1_lowercase: annotation
>         sections_annotator_2_uppercase: annotation
>         sections_annotator_2_lowercase: annotation
> ```
> 
> I'm lost here as well. Fortunately this is the last one.

@magdalenafuentes !

**Comment by rabitt** [comment]
Fixed in #143
